Tasks/AzureResourceGroupDeploymentV2/package.json

```diff
@@ -4,7 +4,7 @@
   "dependencies": {
     "@types/node": "^10.17.0",
     "azure-pipelines-task-lib": "^3.0.6-preview.0",
-    "azure-pipelines-tasks-azure-arm-rest-v2": "1.0.4",
+    "azure-pipelines-tasks-azure-arm-rest-v2": "^2.0.1",
```

Comment:
@ddivanshu have you tested some basic scenarios to ensure that new version of package does not cause any issue? Is there a known list of breaking changes published for this package?

Reply:
Yes , basic scenarios were tested for all the tasks